tests/python/pants_test/reporting/test_reporting.py

```diff
@@ -4,8 +4,17 @@
 
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+import json
+import os
+
+import requests
+from future.backports.urllib.parse import urlencode
```

Comment:
We want to use Py3 first idiom.

```suggestion
from future.moves.urllib.parse import urlencode
```

Reply:
Done